src/js/components/modal.js

```diff
@@ -49,11 +51,15 @@ class Modal {
 
     this.$close.on('click', this.hide.bind(this))
 
-    $(window).on('keyup', this.handler = (e) => {
-      if (e.which === 27) {
-        this.hide()
-      }
-    })
+    this.bindKeyboardListener()
+  }
+
+  bindKeyboardListener () {
+    if (this.options.keyboard) {
+      $(window).on('keyup', this.handler = (e) => {
+        if (e.which === 27) { this.hide() }
```

Comment:
@rands0n @claudiosikeda i'm not sure about using `keys/keyboard.esc` because in this case, we are not checking witch key is, is just to check if you want `keyboard` event to be available.

Maybe change this name to something more meaninfull